packages/code/src/Code/Code.spec.tsx

```diff
@@ -205,7 +205,7 @@ describe('packages/Code', () => {
       const trigger = screen.getByRole('button', { name: 'JavaScript' });
       fireEvent.click(trigger);
 
-      ['Checkmark Icon JavaScript', 'Python'].forEach(lang => {
+      ['JavaScript', 'Python'].forEach(lang => {
```

Comment:
This started failing after I added the aria-label prop to the select option. I'm not sure why it was even passing in the first place because it never had a aria-label